### PR TITLE
Use login for accessing API

### PR DESCRIPTION
**Pull request by joshgachnang**
We dropped the token and the requests were no longer working.

### CODE REVIEW

File: reasonSober/index.js [hunk 1]
```diff
@@ -6,34 +6,32 @@ request = request.defaults({jar: true});
...
+    console.log('Updating reason sober', reasonSoberOptions);
 
-  request(options, callback);
-};
+    request(reasonSoberOptions, (error, response, body) => {
+      if (error) {
+        console.log("Updating reason sober API error: ", error);
+        return;
+      }
+      if (!error && response.statusCode == 200) {
```
Comment (Kurt-von-Laven) [inline reasonSober/index.js]: rm `!error &&`.

Comment (Kurt-von-Laven) [reply reasonSober/index.js]: nit: `==` --> `===`

**Review by Kurt-von-Laven — CHANGES_REQUESTED** [review]
Two minor suggestions

File: reasonSober/index.js (hunk 1 above)
Comment (joshgachnang) [reply reasonSober/index.js]: Done

**Comment by joshgachnang** [comment]
Fixed, PTAL!

**Review by Kurt-von-Laven — APPROVED** [review]
**LGTM**

**Pull request merged**